tests/Microsoft.Bot.Builder.TestBot.WebApi/Global.asax.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.Bot.Builder.TestBot.WebApi
 {
-    public class WebApiApplication : System.Web.HttpApplication
+    public class Global : System.Web.HttpApplication
```

Comment:
I'm not quite sure about renaming this class, as it might be necessary to name it as `WebApiApplication`

Reply:
Done